examples/postfilter1.py

```diff
@@ -15,16 +15,16 @@
 output_dtype = np.dtype(np.float32)
 
 # Set the compression and decompression parameters
-cparams = {"codec": blosc2.Codec.LZ4, "typesize": 4}
-dparams = {"nthreads": 1}
+cparams = blosc2.CParams(codec=blosc2.Codec.LZ4, typesize=4)
+dparams = blosc2.DParams(nthreads=1)
 contiguous = True
 urlpath = None
-storage = {"contiguous": contiguous, "urlpath": urlpath, "cparams": cparams, "dparams": dparams}
+storage = blosc2.Storage(contiguous=contiguous, urlpath=urlpath, mode='a')
 # Remove previous SChunk
 blosc2.remove_urlpath(urlpath)
 # Create and set data
 data = np.arange(200 * 1000 * nchunks, dtype=input_dtype)
-schunk = blosc2.SChunk(chunksize=200 * 1000 * input_dtype.itemsize, data=data, **storage)
+schunk = blosc2.SChunk(chunksize=200 * 1000 * input_dtype.itemsize, data=data, cparams=cparams, dparams=dparams, storage=storage)
```

Comment:
Line is too long.  This could be avoided by triggering ruff (a linter and formatter) on each commit. See the first paragraph in https://github.com/Blosc/python-blosc2/blob/main/README_DEVELOPERS.md